packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -151,7 +151,7 @@ const BUILTIN_KEYWORD_MODIFIERS: Record<string, ModifierDefinitionState> = {
   action: actionModifier,
 };
 
-const BUILTIN_MODIFIERS: Record<string, object> = {
+export const BUILTIN_MODIFIERS: Record<string, object> = {
   ...BUILTIN_KEYWORD_MODIFIERS,
   on,
 };
```

Comment:
This is used in `lookupModifier` on L219. The problem is, it seems like, `(modifier "...")` didn't get implemented in glimmer-vm, or at least when we landed this feature in ember, it didn't, so ember added an AST transform hack to change `(modifier "...")` into `(modifier (-resolve "modifier:..."))`. If that was implemented in glimmer-vm properly like `(component "...")` is, then it should/would have gone through the codepath in L219 and everything should work. That would be the most proper fix.

I was a bit hesitant on fixing it by adding them to the resolver, but I suppose that would be [the most consistent](https://github.com/emberjs/ember.js/blob/be51c93b2b79f5fca8d36db1fcae36a049aeeb25/packages/%40ember/-internals/glimmer/lib/setup-registry.ts#L52-L55). So I suppose why not.

However, I do think we need to fix this properly upstream; the problem with the current AST-transform hack in Ember is it will, as far as I can tell, still do the same AST transform in strict mode templates, which is probably punching a pretty big hole into the entire Embroider/static template universe.